README.md

```diff
@@ -7,7 +7,9 @@ The following Rails app is used as the base project for [The Pivot](http://backe
 Built using Ruby 2.4.1 and Rails 5.1.3
 
 Install required gems:
-`$ bundle install`
+`$ bundle install`  
+
+For image hosting, this application uses Paperclip, which depends on ImageMagick. To install ImageMagick on Mac OS X, you'll want to run the following with [Homebrew] (http://www.brew.sh): `brew install imagemagick `
```

Comment:
Can you change this to `$ brew install imagemagick` so that it's clear it's for the CLI?

Reply:
Great call. Didn't think about it being from the CLI